healpy/fitsfunc.py

```diff
@@ -208,15 +209,19 @@ def write_map(filename,m,nest=False,dtype=np.float32,fits_IDL=True,coord=None,pa
             mm2 = np.asarray(mm)
             cols.append(pf.Column(name=cn,
                                    format='1024%s' % curr_fitsformat,
-                                   array=mm2.reshape(mm2.size/1024,1024),
+                                   array=mm2.reshape(mm2.size//1024,1024),
                                    unit=cu))
         else:
             cols.append(pf.Column(name=cn,
                                    format='%s' % curr_fitsformat,
                                    array=mm,
                                    unit=cu))
 
-    tbhdu = pf.new_table(cols)
+    try:
+        tbhdu = pf.BinTableHDU.from_columns( cols )
```

Comment:
I get the deprecation warnings on the latest PyPI version of astropy: 1.1.2.

On Wed, May 18, 2016 at 12:07 PM, Leo Singer <notifications@github.com>
wrote:

> In healpy/fitsfunc.py
> <https://github.com/healpy/healpy/pull/341#discussion_r63763007>:
>
> >                                     unit=cu))
> >          else:
> >              cols.append(pf.Column(name=cn,
> >                                     format='%s' % curr_fitsformat,
> >                                     array=mm,
> >                                     unit=cu))
> >
> > -    tbhdu = pf.new_table(cols)
> > +    try:
> > +        tbhdu = pf.BinTableHDU.from_columns( cols )
>
> What version of Astropy does this version depend on?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/healpy/healpy/pull/341/files/99b6b3f5ca5cd9145f7aa689ee644cc2a71f16f0#r63763007>
>